docs/api-reference/deck.md

```diff
@@ -113,6 +113,17 @@ A geospatial `viewState` would typically contain the following fields:
 * `pitch` (Number, optional) - Current pitch - used to define a mercator projection if `viewport` is not supplied.
 
 
+#### `initialViewState` (Object)
+
+If `initialViewState` is provided, the `Deck` component will track view state changes from any attached `controller` using internal state, with `initialViewState` as its initial view state.
```

Comment:
The difference in naming between `initialViewState` and `viewState` is not explicit that one makes the map interactive. I'd prefer not to complicate usage by adding yet one more prop if possible - see comment below.